index.js

```diff
@@ -77,6 +82,10 @@ HtmlWebpackIncludeAssetsPlugin.prototype.apply = function (compiler) {
   // Hook into the html-webpack-plugin processing
   compiler.plugin('compilation', function (compilation) {
     compilation.plugin('html-webpack-plugin-before-html-generation', function (htmlPluginData, callback) {
+      if (self.options.files && !castArray(self.options.files).includes(htmlPluginData.outputName)) {
+        return callback(null, htmlPluginData);
+      }
+
```

Comment:
Array.includes does not seem to be supported in older versions of Node (that seems to be why the Travis tests were failing). Could you switch this to `files.indexOf(htmlPluginData.outputName) !== -1` instead ?